Assignment2/toasterdoom-BeñatAgirre.ttl

```diff
@@ -0,0 +1,10 @@
+@base : <http://assignment2.org/#>.
+@prefix : <http://assignment2.org/properties/>.
+
+:Class01            :includes :sensor029,
+                              :Computer101;
```

Comment:
Should finish in .

Reply:
Oops la costumbre. He hecho un force-push y se ha actualizado.